src/main/webapp/app/admin/system-notification-management/system-notification-management.component.ts

```diff
@@ -135,7 +138,10 @@ export class SystemNotificationManagementComponent implements OnInit, OnDestroy
      * Sorts parameters by specified order
      */
     sort() {
-        const result = [this.predicate + ',' + (this.reverse ? 'asc' : 'desc')];
+        const result = [];
+        if (this.predicate) {
+            result.push(this.predicate + ',' + (this.reverse ? 'asc' : 'desc'));
+        }
         if (this.predicate !== 'id') {
             result.push('id');
         }
```

Comment:
[nit] Is this check related to the check above? Should we push `id` even if `this.predicate` is undefined?

Reply:
This is a good question I am not sure. But because Id should be set if  "predicate" is not equal to "id" it is also the case now because predicate = undefined -> not equal to id. If I understood it correctly it did correctly filter by id on the server so I think this is fine.